pkg/controller/v1alpha1/inferencegraph/controller.go

```diff
@@ -144,13 +144,14 @@ func (r *InferenceGraphReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			if route.ServiceName != "" {
 				err := r.Client.Get(ctx, types.NamespacedName{Namespace: graph.Namespace, Name: route.ServiceName}, &isvc)
 				if err == nil {
-					if isvc.Status.Address != nil && isvc.Status.Address.URL != nil {
-						if graph.Spec.Nodes[node].Steps[i].ServiceURL == "" {
-							graph.Spec.Nodes[node].Steps[i].ServiceURL = isvc.Status.Address.URL.String()
+					if graph.Spec.Nodes[node].Steps[i].ServiceURL == "" {
```

Comment:
@rachitchauhan43 can you take a look ? We removed the path from the address.url on inference status as path should be specified by user when calling the service. So now we need to set a default path for inference graph if serviceUrl is not specified, the behavior did not change.